app/portainer/components/environment-variables-panel/environment-variables-simple-mode/environment-variables-simple-mode.html

```diff
@@ -6,6 +6,15 @@
     <pr-icon icon="'alert-circle'" mode="'primary'"></pr-icon>
     Switch to advanced mode to copy & paste multiple variables
   </div>
+  <div class="col-sm-12 form-inline env-items-list">
```

Comment:
why is this change needed? I looked for an issue with env-vars panel in the jira but couldn't find any